### ISSUE TITLE
Training epochs

### ISSUE THREAD

**Issue by BounharAbdelaziz** [chaofengc/Face-SPARNet#11]
Hi, I wanna first thank you for sharing such project! 

I wonder if you need to add +1 in the training loop (train.py line 27). In fact as stated in the paper, **SPARNet-HD was trained over 10 epochs**, which is something we can see in the train.sh file (--total_epochs 10). I think you do it that way, it will train over 11 epochs, I wonder if you saw it before (not sure though since you saw the pcp loss bug)

Can you please confirm that? Many thanks in advance!

With best,

**Comment by BounharAbdelaziz** [comment]
Thank you very much for your instant reply !
